src/App.js

```diff
@@ -1,10 +1,26 @@
-
+import styled from "styled-components";
```

Comment:
pliki reactowe powinny (ze względu na dobre praktyki) być plikami 'jsx'. 
Generalnie dla aplikacji nie ma to znaczenia, ale składnia JSX z której korzystamy w React nie jest czystym JSem, dlatego też dla rozróżnienia (mimo braku różnic) powinno się używać JSX właśnie.
Tutaj większy opis: 
https://github.com/airbnb/javascript/pull/985